invenio_rdm_migrator/load/postgresql.py

```diff
@@ -45,7 +45,13 @@ class PostgreSQLCopyLoad(Load):
     """PostgreSQL COPY load."""
 
     def __init__(
-        self, db_uri, data_dir, table_generators, existing_data=False, **kwargs
+        self,
+        db_uri,
+        table_generators,
+        tmp_dir=None,
+        data_dir=None,
```

Comment:
minor/comment: just to clarify:

- `tmp_dir` is used now when table generators actually populate a CSV file, and
- `data_dir` is for when we have existing data?

Reply:
Yes, correct. `data_dir` is seen as permanent, a place where actual data to import is placed. `tmp_dit` is _temporary_, which you can remove after a run if you are not happy with the result for whatever reason.